convert2rhel/unit_tests/checks_test.py

```diff
@@ -813,3 +825,86 @@ def test_custom_repos_are_invalid(self):
         self.assertEqual(len(checks.logger.critical_msgs), 1)
         self.assertEqual(len(checks.logger.info_msgs), 0)
         self.assertTrue("Unable to access the repositories passed through " in checks.logger.critical_msgs[0])
+
+
+@pytest.mark.parametrize(
+    ("package", "return_code", "raise_system_exit"),
+    (
+        (
+            "convert2rhel.noarch   0.24-1.20211019110627581338.pr343.11.g4ff8753.el8   copr:copr.fedorainfracloud.org:group_oamg:convert2rhel\n"
+            "convert2rhel.src      0.24-1.20211019110627581338.pr343.11.g4ff8753.el8   copr:copr.fedorainfracloud.org:group_oamg:convert2rhel\n"
+            "tzdata.noarch         2021c-1.el8                                         baseos\n",
```

Comment:
Add this as well since Toshio mentioned it can appear
```
Security: kernel-core-5.14.15-100.fc33.x86_64 is an installed security update
Security: kernel-core-5.14.13-100.fc33.x86_64 is the currently running version
```

Reply:
@SpyTec, Those two lines are just informational and not telling the end-user that `kernel-core` needs to be updated? So our solution should gracefully ignore that, right?

Reply:
Yeah exactly. We should still have it in the test to make sure we don't pick up on it though in the function

Reply:
Added a few more cases to the test as well to the addition of the two lines of the `Security:`log